### DISCUSSION TITLE
Help me understand adminLTE dark mode concept

### DISCUSSION THREAD

**Discussion by brendan-pike** [ColorlibHQ/AdminLTE#5658] [Q&A]
I've enjoyed building on adminLTE however I've found that it does not use or browser/user preference detection color-scheme
which then means we can't make use of  @media (prefers-color-scheme) in our css and correct dark support for things like html  <picture> ie. https://dev.to/ziratsu/switch-to-a-darker-image-when-on-dark-mode-2lkh

Where as adminLTE relies on body.dark-mode so I'm trying to understand why its been implemented this way and if there is a way for the two to co-exist?

**Comment by danny007in** [comment]
Check the progress and updates for AdminLTE v4 release [here](https://github.com/ColorlibHQ/AdminLTE/issues/3692).
